api.go

```diff
@@ -32,9 +31,15 @@ const (
 	API_ERR_INTERNAL
 )
 
+type deploymentsResult struct {
+	deployments []DataDeployment
+	err         error
+}
+
 var (
-	deploymentsChanged = make(chan string)
-	addSubscriber      = make(chan chan string)
+	deploymentsChanged = make(chan interface{})
```

Comment:
It would be better if you make `deploymentsChanged` a buffered channel. Otherwise, in `debounce()`, either a blocking in `in` or `out` will likely affect the other.
`deploymentsChanged = make(chan interface{}, IN_BUFFER_SIZE)`